src/components/ContentSection.tsx

```diff
@@ -9,136 +8,47 @@ export type contentSectionData = {
-          <motion.h3
-            whileInView={{
-              x: 0,
-              opacity: 1,
-              transition: {
-                duration: 0.5,
-              },
-            }}
-            initial={{ opacity: 0, x: -10 }}
-            className={`mb-4 pt-10 text-3xl text-black lg:text-5xl`}
-          >
-            {props.title}
-          </motion.h3>
-
-          {/* Paragraph with delayed fade and slide-in animation */}
-          <motion.p
-            initial={{ opacity: 0, x: -10 }}
-            whileInView={{
-              opacity: 1,
-              x: 0,
-              transition: {
-                duration: 0.5,
-                delay: 0.2,
-              },
-            }}
-            className={`
-              font-raleway
-              mb-8
-              text-lg
-              text-[#757371]
-             
-            `}
-          >
-            {props.description}
-          </motion.p>
-
-          {/* Conditional rendering for the call-to-action button with animation */}
-
-          {props.action ? (
-            <motion.span
-              initial={{ opacity: 0, x: -10 }}
-              whileInView={{
-                opacity: 1,
-                x: 0,
-                transition: {
-                  duration: 0.5,
-                  delay: 0.4,
-                },
-              }}
-            >
-              <Link
-                href={props.action}
-                className={`
-                    font-raleway
-                    rounded-md
-                    border
-                    border-black
-                    px-8
-                    py-3
-                    transition
-                    duration-150
-                    ease-in-out
-                    hover:border-[#A87133]
-                    hover:bg-[#A87133]
-                    hover:text-white
-                  `}
-              >
-                {props.actionTitle}
-              </Link>
-            </motion.span>
-          ) : null}
-          <br />
-          <br />
+      <div className={`flex flex-col ${props.reverse ? "lg:flex-row" : "lg:flex-row-reverse"} gap-6 lg:items-start lg:mt-0 mt-10`}>
+        <div className="w-full px-10 md:px-20 lg:py-12 lg:px-32 sticky lg:top-1">
+          <h3 className="text-5xl mb-4 font-light font-amiora capitalize">{props.title}</h3>
+          {renderDescription(props.description)}
+          {
+            props.action && (
+              <>
+                <Link href={props.action} className="border border-primary rounded-md px-6 py-2 mt-4 text-xl font-raleway text-primary hover:bg-primary hover:text-white transition-colors duration-200">
+                  {props.actionTitle}
+                </Link>
+                <br />
+                <br />
+              </>
+            )
+          }
+        </div>
+        <div className={`w-full scroll-up h-72 md:h-96 lg:h-[95vh] xl:h-[85vh] overflow-hidden`}>
```

Comment:
so we use nextjs Image component, the reason is to avoid Cumulative Layout Shift and Image optimization. ukitumia hii peke yake bila specifying width and height itakua shida